CMakeLists_files.cmake

```diff
@@ -275,6 +275,8 @@ if(ENABLE_ECL_OUTPUT)
           src/opm/output/eclipse/RestartValue.cpp
           src/opm/output/eclipse/WriteInit.cpp
           src/opm/output/eclipse/WriteRFT.cpp
+          src/opm/output/eclipse/WriteRPT.cpp
+          src/opm/output/eclipse/WriteRPT_WELSPECS.cpp
```

Comment:
I suggest you create a directory `..eclipse/report` and then a file `Welspecs.cpp` there

Reply:
👍